BAZProjectC1/Services/ApiServiceRequest.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ApiServiceRequest.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 15/09/22.
+//
+
+import Foundation
+
+public class ApiServiceRequest{
```

Comment:
Nitpick: Deja un espacio entre el nombre de tu clase y las llaves
```suggestion
public class ApiServiceRequest {
```